src/libraries/System.Net.Http/tests/StressTests/HttpStress/Dockerfile

```diff
@@ -43,7 +42,7 @@ ENV STRESS_ROLE=''
 ENV STRESS_ARGS=''
 
 # configure adress sanitizer
-# ENV ASAN_OPTIONS='detect_leaks=0'
-# ENV LD_PRELOAD=/usr/lib/gcc/x86_64-linux-gnu/12/libasan.so
+ENV ASAN_OPTIONS='detect_leaks=0'
```

Comment:
You should grab all of the options we return from `__asan_default_options` in the minipal. That has the complete list of the options required to use ASAN with CoreCLR.

Reply:
We are building msquic in this Dockerfile. CoreCLR is built without ASAN for HttpStress.

Reply:
Yes, but the ASAN_OPTIONs along with the LD_PRELOAD influence the whole process, so the options incompatible with CoreCLR should be disabled.